### PR TITLE
Feature/datasets/add new datasets to api

### PR DESCRIPTION
**Pull request by leothomas**
### What I did
Added 2 new datasets: `recovery` and `slowdown` + updated some of the dataset metadata files to correctly mirror the [metadata files](https://github.com/NASA-IMPACT/covid-dashboard/tree/develop/app/assets/scripts/components/common/layers) currently in use in production. 

### How I did it
- Added/modified metadata files in `/datasets`
- Added `compare` and `paint: raster_opacity` fields to the Datasets Model in `static.py`

### How you can test it
Pull this branch and run `docker-compose up --build` and hit the `/datasets` endpoint at `http://localhost:8000/v1/datasets`. 

For reference, I'm attaching the [dataset names and domains returned by the endpoint](https://github.com/NASA-IMPACT/covid-api/files/5490700/datasets_endpoint_output.txt) for each spotlight (the last spotlight, `""`) corresponds to a LIST operation on `/datasets` and therefore represents all datasets. 

Some notes: 
- It seems that `togo-agriculture` dataset has no domain. Is that the desired behaviour? 
- More files have been delivered to `s3://covid-eo-data/agriculture/` whereas to keep with the convention that each dataset should reside under a folder corresponding to the dataset's id, those files should have been stored under `s3://covid-eo-data/agriculture-cropmonitor/` - additionally since these files are delivered monthly, I've changed their timestamp format from `YYYY-MM-DD` to `YYYYMM`.

### CODE REVIEW

File: covid_api/db/static/datasets/co2.json [hunk 1]
```diff
@@ -33,7 +34,7 @@
         "source": {
             "type": "raster",
             "tiles": [
-                "{api_url}//{z}/{x}/{y}@1x?url=s3://covid-eo-data/xco2/xco2_15day_base.{date}.tif&resampling_method=bilinear&bidx=1&rescale=0.000408%2C0.000419&color_map=rdylbu_r&color_formula=gamma r {gamma}"
+                "{api_url}/{z}/{x}/{y}@1x?url=s3://covid-eo-data/xco2/xco2_15day_base.{date}.tif&resampling_method=bilinear&bidx=1&rescale=0.000408%2C0.000419&color_map=rdylbu_r&color_formula=gamma r {gamma}"
```
Comment (olafveerman) [inline covid_api/db/static/datasets/co2.json:37]: We are using the 16 day product.

```
`${config.api}/{z}/{x}/{y}@1x?url=s3://covid-eo-data/xco2-base/xco2_16day_base.{date}.tif&resampling_method=bilinear&bidx=1&rescale=0.000408%2C0.000419&color_map=rdylbu_r&color_formula=gamma r {gamma}`
```

and

`${config.api}/{z}/{x}/{y}@1x?url=s3://covid-eo-data/xco2-mean/xco2_16day_mean.{date}.tif&resampling_method=bilinear&bidx=1&rescale=0.000408%2C0.000419&color_map=rdylbu_r&color_formula=gamma r {gamma}`

File: covid_api/db/static/datasets/co2-diff.json [hunk 2]
```diff
@@ -24,7 +26,7 @@
         "detections-ship",
         "detections-plane"
     ],
-    "enabled": true,
+    "enabled": false,
```
Comment (olafveerman) [inline covid_api/db/static/datasets/co2-diff.json:29]: All CO2 layers are using the 16 day product:

```
${config.api}/{z}/{x}/{y}@1x?url=s3://covid-eo-data/xco2-diff/xco2_16day_diff.{date}.tif&resampling_method=bilinear&bidx=1&rescale=-0.000001%2C0.000001&color_map=rdbu_r
```

**Review by olafveerman — CHANGES_REQUESTED** [review]
Left some comments on the CO2 product, but other than that looks good.

File: covid_api/db/static/datasets/co2.json (hunk 1 above)
Comment (leothomas) [reply covid_api/db/static/datasets/co2.json]: Done

File: covid_api/db/static/datasets/co2-diff.json (hunk 2 above)
Comment (leothomas) [reply covid_api/db/static/datasets/co2-diff.json:29]: Done

**Review by olafveerman — APPROVED** [review]

**Comment by olafveerman** [comment]
Looks good. Ready to merge @leothomas ?

**Pull request merged**